packages/user/src/components/Invitation/InvitationFormFields.tsx

```diff
@@ -76,11 +90,11 @@ export const InvitationFormFields: React.FC<IProperties> = ({
           name="app"
           label={t("invitation.form.app.label")}
           placeholder={t("invitation.form.app.placeholder")}
-          options={apps}
+          options={updatedApps}
         />
       ) : null}
 
-      {filteredRoles?.length ? (
+      {apps?.length ? (
```

Comment:
we want to be able to display rolePicker if the roles prop is provided. With this, we may lose this ability. 
So 
- if apps prop is provided, use roles options from app.supportedRoles
- if apps is not given and instead roles  prop is given, display role picker with roles options, without app picker.